test/python/fe_api/sdpa/test_sdpa_bwd_sm80.py

```
@@ -204,15 +206,146 @@ def test_sdpa_bwd_sm80_d64_fast_path():
+    from cudnn.sdpa.fwd import sdpa_fwd_wrapper_sm80
+
+    t = int(sum(lens))
+    cu = torch.tensor([0] + list(itertools.accumulate(lens)), dtype=torch.int32, device="cuda")
+    q = torch.randn(1, t, h, d_qk, dtype=torch.float16, device="cuda")
+    k = torch.randn(1, t, h, d_qk, dtype=torch.float16, device="cuda")
+    v = torch.randn(1, t, h, d_v, dtype=torch.float16, device="cuda")
+    do = torch.randn(1, t, h, d_v, dtype=torch.float16, device="cuda")
+    fwd = sdpa_fwd_wrapper_sm80(q, k, v, is_causal=True, cum_seqlen_q_tensor=cu, cum_seqlen_k_tensor=cu, max_s_q=int(max(lens)))
+    out = sdpa_bwd_wrapper_sm80(q, k, v, fwd["o_tensor"], do, fwd["lse_tensor"], is_causal=True, cum_seqlen_q_tensor=cu, cum_seqlen_k_tensor=cu)
+    if check_grads:
+        for i, s_len in enumerate(lens):
```

Ruff (0.16.2)

[warning] Loop control variable `s_len` not used within loop body

Rename unused `s_len` to `_s_len`

(B007)